package TestNGExecution;
import org.testng.annotations.Test;

public class ThrowThrowsException {
@Test
	public void AException() {		
		int a =10/0; //Arithmetic Exception
	}

}
